requires.py

```diff
@@ -32,11 +32,11 @@ def changed(self):
         conv.toggle_state('{relation_name}.ready',
                           active=self.is_spark_started())
 
-    @hook('{requires:spark}-relation-departed')
+    @hook('{provides:spark}-relation-departed')
```

Comment:
i think you mean `requires` instead of `provides` here, since we're in requires.py.